Remove preview tag from Risky IP Workbook

zhvolosh opened the pull request:
Publish for customers by removing preview tag + change name of Sign-In Analysis to display AAD and ADFS information

## PR Checklist

* [X] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

Removing the preview tag from the Risky IP Report Workbook to have it be available to customers + changing name of Sign-Ins Workbook to show AAD+ADFS title. 

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

zhvolosh commented:
@microsoft/azure-ad-workbooks can someone take a look and merge this request? I'm looking to release this Workbook to customers without the preview tag after validating in the past Q.

SarahBar reviewed (changes requested):
Before merging the Risky IP Workbook PR, could you do a few things?

(1) Add the explanation at the beginning to call out the need for ADFSSignInLogs stream and how to make it available for Workbooks to consume.

(2) In the "Risky IP Count by time" chart, make the y axis count only whole numbers from 1 (1, 2, 3...). Currently, it shows decimal increments, which doesn't make sense because you can't have a fraction of an IP address counted.

(3) Optional but recommended: Add more filtering options to the tables--for example, IP Address, Location, User Name, User ID, Error Code and/or Error Description. You can choose the ones that you think make the most sense given what users are likely to be searching for. Especially in the case that a table can only return a limited number of results, the filters make it possible to find something that would otherwise be hidden.

zhvolosh commented:
Addressed the following:
1. Added a toggle tab to "Show Help" with prerequisite details to enable Log Analytics.

2. I think this one cannot be changed to not have decimals as the 3 options are decimal, percent, and currency. I played around with the settings and there is a limit where I cannot set a value of zero for the minimum significant digits. If the range is bigger, and not all the values are 1, it should shift to be correct since it is automatic.
![image](https://user-images.githubusercontent.com/22526610/135329474-5d9c3ba4-babd-4670-81ae-79262e8f4bea.png)

3. I added User Name to the Select By IP Table, which would also make sense in the filtration.
